build-repo.sh

```diff
@@ -17,78 +17,41 @@
 
 set -xe
 
-mkdir -p binaries
-
-export PROVIDE_ONLY=true
-export SUDO_CMD=sudo
-export BASE_PATH="binaries/"
-export APTLY_ONLY="focal"
-
 # Aptly does not need sudo, as the jenkins user is in the aptly group
 
-if [ -f multidist.buildinfo ]; then
-	echo "Doing multibuild"
-	MULTI_DIST=$(cat multidist.buildinfo)
-  for t in multidist*.tar.gz ; do
-    tar --overwrite -xvzf $t
-  done
-	rm multidist*.tar.gz || true
-  export rootwp=$(pwd)
+push_aptly() {
+    local release="$1"
+    local path="$2"
```

Comment:
`local` technically isn't in POSIX sh. Consider changing shebang to `/bin/dash`?

Reply:
Fixed, changed to bash

Reply:
I wrote `dash` not `bash` (`dash` does support `local`). But `bash` works too I guess (and maybe more portable? but you still need cowbuilder...). The startup speed gain is completely drowned out by other program's execution speed.